lib/cardano-sl.cabal

```diff
@@ -60,6 +59,8 @@ library
                         Pos.Behavior
 
                         -- Something
+                        Pos.Types
```

Comment:
Why did you move it here? 🤔 
They were alphabetically sorted (or very close to be sorted)

Reply:
Because `-- Txp` was a wrong place for it